tests/routes_tests/test_auth.py

```diff
@@ -91,7 +91,7 @@ def check_redirect_contains_redirect_uri(response, expected_url):
     # which makes a double slash like 'http://localhost//authorize'
     # which won't work. So testing to make sure there is no '//'
     # other than '://'
-    assert re.search("[^:]\/\/", redirect_uri) is None
+    assert re.search("[^:]//", redirect_uri) is None
```

Comment:
are we sure about this change? This doesn't turn this into a regexp matching?

Reply:
Seems to have the same behavior? But I'm not sure exactly what you mean by "turn this into a regexp matching"

```python
>>> import re
>>> re.search("[^:]//", "localhost://test")
>>> print(re.search("[^:]//", "localhost://test"))
None
>>> print(re.search("[^:]\/\/", "localhost://test"))
None
>>> print(re.search("[^:]//", "localhost://test//"))
<re.Match object; span=(15, 18), match='t//'>
>>> print(re.search("[^:]\/\/", "localhost://test//"))
<re.Match object; span=(15, 18), match='t//'>

```